### PR TITLE
Run autoformatting on all rule source

### PR DESCRIPTION
**Pull request by rw-access**
Before I make a few other PRs like #671 where I'll use autoformatting, I figure we can run it first on all rules. If we find anything we don't like, we can manually fix/revert it or add more conditions to the MQL autoformat command.

### CODE REVIEW

File: detection-rules/attachment_dropbox_image_suspicious_links.yml
Comment (morriscode) [inline detection-rules/attachment_dropbox_image_suspicious_links.yml:1]: Do we want to pull JPEG here and replace it with gif? Or should all these be converted to the $file_types_images?

File: detection-rules/attachment_any_html_new_sender.yml
Comment (morriscode) [inline detection-rules/attachment_any_html_new_sender.yml:1]: This is probably personal preference and readability, but the single line or condition might be lost or at the very least isn't as apparent as putting it on a second line.

File: discovery-rules/attachment_suspicious_macro.yml
Comment (morriscode) [inline discovery-rules/attachment_suspicious_macro.yml:1]: Changing the comma delimited lists to from a /s to a \r\n makes the rule bigger but imo easier to read.

**Review by morriscode — APPROVED** [review]
This looks great, I left a few comments that are probably just personal preferences or things i'm accustom to, but in review I'm really excited to have a level of consistent formatting throughout the feed rules.

File: discovery-rules/attachment_suspicious_macro.yml
Comment (rw-access) [reply discovery-rules/attachment_suspicious_macro.yml:1]: For sure. I've also thought about having them lay out in a grid. Like for example, you could do 4 columns per line, all lined up vertically. It's a little trickier but could be neat.

And after a certain size, it makes sense to put them in a list.

**Pull request merged**